pages/common/parallel.md

```diff
@@ -26,3 +26,11 @@
 - Run on multiple machines via SSH:
 
 `parallel -S {{machine1}},{{machine2}} {{command}} ::: {{arg1}} {{arg2}}`
+
+- Download 4 files simultaneously from a text file containing links showing progress:
+
+`parallel -j4 --bar --eta wget -q {} :::: links.txt`
+
+- Print the jobs parallel is running in stderr:
```

Comment:
```suggestion
- Print the jobs parallel is running in `stderr`:
```
We generally enclose terms like "stdin" and "stderr" inside backticks (this would make it consistent with the above descriptions too).